### PR TITLE
Implement get_device_info for F5359 used by KPN Box 12 using single REST call

### PR DESCRIPTION
**Pull request by rhpijnacker**
Follow up of #108, improved to do just one API call to get the device info.

### CODE REVIEW

**Comment by rhpijnacker** [comment]
@iMicknl Could you send me a copy of the device info as reported by your device?
Then I can check whether I'm picking the right properties.

**Comment by iMicknl** [comment]
> @iMicknl Mick Vleeshouwer FTE Could you send me a copy of the device info as reported by your device? Then I can check whether I'm picking the right properties.

Sorry, I am not able to test it for a while since I don't own the device myself. I am happy to merge this in and later on see where we can make changes.

File: sagemcom_api/client.py [hunk 1]
```diff
@@ -341,29 +362,20 @@ async def get_device_info(self) -> DeviceInfo:
...
+                }
             )
-            device_info.serial_number = await self.get_value_by_xpath(
-                "Device/DeviceInfo/SerialNumber"
-            )
-            device_info.software_version = await self.get_value_by_xpath(
-                "Device/DeviceInfo/SoftwareVersion"
-            )
-            return device_info
+            data["manufacturer"] = "Sagemcom"
```
Comment (iMicknl) [inline sagemcom_api/client.py:376]: The only addition is that my device has `Sagemcom` as manufacturer.

**Comment by iMicknl** [comment]
@rhpijnacker I reserved some time this weekend to fix everything and finally publish. Sorry for the delays :(

**Comment by iMicknl** [comment]
> @iMicknl Could you send me a copy of the device info as reported by your device?

Are you on (HA) Discord as well? I can share my device JSON, however there are some hostnames / mac addresses in as well, that I would rather share privately. 

I just pushed some updates to have CI/CD ready to push everything faster forward.

**Comment by rhpijnacker** [comment]
> Are you on (HA) Discord as well? 

No, sorry.
Feel free to "blur" the hostnames and mac addresses, though

**Comment by iMicknl** [comment]
Will do, it is just that it is a lot of work and I don't have a lot of time to spend on this integration.

**Pull request merged**